exercise 22/index.js

```diff
@@ -0,0 +1,8 @@
+alert ('Enter two numbers to see if the first is greater than the second' )
+let a=prompt("enter a value" ), b = prompt("enter a value"); 
+if(a<b)
+alert('a<b');
+else if (a>b)
+alert ('a>b');
+else (a==b)
+alert ('a=b');
```

Comment:
1. It's better to define each variable from new line using new let. It's just more clear and less-error way.
2. You need to convert prompt values to "number" type, before comparing them, otherwise after prompt they are "string" type by default.
3. If you are using if, else if structure, you need to write it using curly braces
`if(condition1) {
//statement1
} else if (condition2) {
//statement2
} else {
//statement 3
}`
4. Last `else` can not have condition `else(condition)` is incorrect. `else if(condition)` can have condition.